.github/workflows/android-build.yml

```
@@ -0,0 +1,55 @@
+name: Android Debug Build
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)